src/resources/scripts/countRequests.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+nbrVisits=0
+nbrActions=0
+
+while IFS= read -r line; do
+    # ignore bot and security check requests
+    bot="[bB]ot|OpenVAS-VT";
+    if [[ "$line" =~ $bot ]]; then
+        continue;
+    fi
+    
+    browser="Chrome|Edg|Safari|OPR|Firefox";
+    if [[ "$line" =~ $browser ]]; then
+    # filter http code (2xx) by using regex of ' <http code> XY ' 
+      code="[[:space:]]2[[:digit:]]{2}[[:space:]][[:digit:]]+[[:space:]]"
+      if [[ "$line" =~ "GET" && $line =~ $code ]]; then
+        ((nbrVisits++))
+      requests="PUT|POST|DELETE"
+      elif [[ "$line" =~ $requests ]]; then
+        ((nbrActions++))
+      fi
+    fi
+done < $1
+
+res="{\"visits\": $nbrVisits, \"actions\": $nbrActions}"
+
+curl -X POST -H "Authorization: Bearer <token>" -F "value=$res" https://biigle.de/api/v1/kpis -v
```

Comment:
This is how you can send a JSON payload that won't be treated as plain string by Laravel:

```suggestion
curl -X POST \
   -H "Authorization: Bearer <token>" \
   -H "Content-Type: application/json" \
   -d "{\"visits\": $nbrVisits, \"actions\": $nbrActions}" \
   https://biigle.de/api/v1/kpis
```

The request above will have the `visits` and `actions` parameters, so the request validation could be done like this:

```php
['actions' => 'required|integer', 'visits' => 'required|integer']
```

And the values can be accessed like this:

```php
$visits = $request->input('visits');
```